interfaces/HO-Portal/src/app/users/users.page.scss

```diff
@@ -0,0 +1,40 @@
+.tabs {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  margin-top: 30px;
+  padding-left: 32px;
+  border-bottom: 1px solid var(--fiveten-neutral-600);
+}
+
+.tab {
+  --background: #f4f7fa;
+  margin-right: 16px;
+  --color-checked: #4f22d7;
+}
```

Comment:
These "tab"-styles are already defined somewhere else right?

Or we need to make them reusable on other pages.

Reply:
hmmm not really because this is from the new design. But ok, will put it in global.scss.